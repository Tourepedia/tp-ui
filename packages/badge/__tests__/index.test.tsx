import * as React from "react"
import * as renderer from "react-test-renderer"
import Badge from "./../src/index"

describe("renders", () => {
  it("with a default type of button", () => {
    const wrapper = renderer.create(<Badge>10</Badge>)
    expect(wrapper.toJSON()).toMatchSnapshot()
  })
})
