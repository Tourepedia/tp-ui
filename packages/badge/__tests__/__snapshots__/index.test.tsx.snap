// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with 10 1`] = `
<span
  style={
    Object {
      "background": "pink",
      "border": "1px solid silver",
      "borderRadius": "9em",
      "padding": "5px 10px",
    }
  }
>
  10
</span>
`;
